.dev/CRAN_Release.cmd

```diff
@@ -232,6 +232,23 @@ require(data.table)
 test.data.table()
 q("no")
 
+# test no partial matching errors in examples
+R
+options(
+  warn = 2L,
+  warnPartialMatchArgs = TRUE,
+  warnPartialMatchAttr = TRUE,
+  warnPartialMatchDollar = TRUE
+)
+for (f in list.files('man', full.names = TRUE)) {
+    l = readLines(f)
+    if (!any(grepl(r'(\examples)', l, fixed=TRUE))) next
+    nm = gsub(r'(\\name\{([^}]*)\})', '\\1', grep(r'(\\name)', l, value=TRUE))
```

Comment:
this `r'(\\name\{([^}]*)\})'` requires R 4.0.0 so stopifnot would be useful just before entering the loop

Reply:
hmm I wasn't worried about that since this is in CRAN_release script, hopefully that's run on latest release

Reply:
Yeah, that is no problem but to avoid confusion if reached here on older R, it is good to have stopifnot somewhere.
I am still on 3.6.3, will move to 4.0.1 directly.